src/raftstore/store/util.rs

```diff
@@ -376,6 +394,17 @@ pub fn get_region_approximate_size(db: &DB, region: &metapb::Region) -> Result<u
 
 /// Get the approximate number of keys in the region.
 pub fn get_region_approximate_keys(db: &DB, region: &metapb::Region) -> Result<u64> {
+    // try to get from RangeProperties first.
+    match get_region_approximate_keys_cf(db, CF_WRITE, region) {
```

Comment:
Should we get from both CF_DEFAULT and CF_WRITE and choose the bigger one?

Reply:
for small data like index, mostly it isn't in the default CF. mostly, write CF has more keys except using RawKV.

Reply:
Since we already collect the properties, I think we can support RawKV later when necessary.